cmd/kubebuilder/create/resource/run.go

```diff
@@ -63,6 +66,7 @@ func AddCreateResource(cmd *cobra.Command) {
 	createResourceCmd.Flags().BoolVar(&controller, "controller", true, "if true, generate the controller code for the resource")
 	createResourceCmd.Flags().BoolVar(&generate, "generate", true, "generate source code")
 	createResourceCmd.Flags().BoolVar(&createutil.AllowPluralKind, "plural-kind", false, "allow the kind to be plural")
+	createResourceCmd.Flags().BoolVar(&skipRBACValidation, "skip-rbac-validation", false, "if set to true, skip validation for RBAC annotations")
```

Comment:
I'm thinking if we can change `&skipRBACValidation` to `&generatedcmd.SkipRBACValidation`. Then we only need to define the flag once in `cmd/kubebuilder/generate/generate.go` or other better location and use it in different places.

Reply:
If we do that, it will tightly couple the implementation of `generate` pkg with this pkg which isn't desirable at this stage.

Reply:
Fair enough. I think the change is good. We can merge it.